repo_policy_compliance/blueprint.py

```diff
@@ -206,3 +207,13 @@ def workflow_dispatch_check_run(body: WorkflowDispatchInput) -> Response:
         return Response(response=report.reason, status=403)
 
     return Response(status=204)
+
+
+@repo_policy_compliance.route(HEALTH_ENDPOINT, methods=["GET"])
+def health() -> Response:
+    """Health check endpoint.
+
+    Returns:
+        204 response.
+    """
+    return Response(status=204)
```

Comment:
Since the application interacts with Github API shouldn't we try a "no-op" operation on github (that'll ensure for example that the token is properly configured) ?

Reply:
Good idea, added

Reply:
GitHub API has rate limits, if we use the same token for multiple applications/units, there will be risk of exceeding the rate limits and causing normal operations to fail.

Reply:
For personal tokens it seems to be 5000 requests per hour per token. I think on production we're not using a personal token and instead an organisation token which should have higher limits, though I don't know the specific docs. In terms of the overall number of requests, checking a single job takes on the order of 10 API requests, so this will be a minor addition. I wouldn't expect this endpoint to be used a lot, just after, e.g., a restart to check that everything is ok

Reply:
Maybe we should change the endpoint name, since health check sounds like something you would put in the docker health check or Kubernetes health check and it would run repeatedly every minutes or so.

Reply:
Any suggestions? I can see the reason if this was a charm, this is more a library so in a different context?